manager.go

```diff
@@ -640,7 +640,7 @@ func buildPullSpec(namespace, tagName string) string {
 	if strings.HasPrefix(tagName, "sha256:") {
 		delimiter = "@"
 	}
-	return fmt.Sprintf("registry.svc.ci.openshift.org/%s/release%s%s", namespace, delimiter, tagName)
+	return fmt.Sprintf("registry.ci.openshift.org/%s/release%s%s", namespace, delimiter, tagName)
```

Comment:
This is not safe and we are going to have to remove it.  WHo uses this code?

I would drop this for now.

Reply:
Drop from this PR, move to another.

Reply:
Unfortunately, it looks like this logic is indirectly used by a majority of the slack commands.

Reply:
Hrm.  I guess we can deal with it for now.